src/intl/en/page-wallets-find-wallet.json

```diff
@@ -1,142 +1,92 @@
+  "page-find-wallet-ens-support": "ENS support",
+  "page-find-wallet-ens-support-desc": "Wallets that support Ethereum Name Service (ENS)",
+  "page-find-wallet-token-importing": "Token importing",
+  "page-find-wallet-token-importing-desc": "Import any ERC-20 token to use in the wallet",
+  "page-find-wallet-fee-optimization": "Fee optimization",
+  "page-find-wallet-fee-optimization-desc": "Support type 2 transactions for optimized gas fees, and fee refunds for unused gas",
+  "page-find-wallet-buy-crypto": "Buy crypto",
+  "page-find-wallet-buy-crypto-desc": "Buy crypto with fiat directly in the wallet \n *Note: buying crypto may be region specific",
+  "page-find-wallet-sell-for-fiat": "Sell for fiat",
+  "page-find-wallet-sell-for-fiat-desc": "Sell crypto to fiat directly in the wallet \n *Note: withdrawing crypto may be region specific",
+  "page-find-wallet-multisig": "Multisig",
+  "page-find-wallet-multisig-desc": "Wallets that require multiple signatures to authorize a transaction",
+  "page-find-wallet-social-recovery": "Social recovery",
+  "page-find-wallet-social-recovery-desc": "Wallets that allow guardians to change the signing key for smart contract wallets",
+  "page-find-wallet-token-support": "Token support",
+  "page-find-wallet-features": "Features",
+  "page-find-wallet-security": "Security",
+  "page-find-wallet-smart-contract": "Smart contract",
+  "page-find-wallet-check-out": "Check out",
+  "page-find-wallet-info-updated-on": "info updated on",
+  "page-find-wallet-showing-all-wallets": "Showing all wallets",
+  "page-find-wallet-showing": "Showing",
+  "page-find-wallet-wallets": "wallets",
+  "page-find-wallet-iOS": "iOS",
+  "page-find-wallet-android": "Android",
+  "page-find-wallet-linux": "Linux",
+  "page-find-wallet-macOS": "macOS",
+  "page-find-wallet-windows": "Windows",
+  "page-find-wallet-chromium": "Chromium",
+  "page-find-wallet-firefox": "Firefox",
+  "page-find-wallet-hardware": "Hardware",
+  "page-find-wallet-hardware-desc": "Hardware wallets",
+  "page-find-wallet-new-to-crypto-title": "New to crypto",
+  "page-find-wallet-new-to-crypto-desc": "You are a first time user looking for your first wallet",
+  "page-find-wallet-nfts-title": "NFTs",
+  "page-find-wallet-nfts-desc": "You are someone that is all about NFTs and want a wallet with NFT support",
+  "page-find-wallet-hodler-title": "Hodler",
+  "page-find-wallet-hodler-desc": "You are someone that has tokens and don’t want to touch them",
+  "page-find-wallet-finance-title": "Finance",
+  "page-find-wallet-finance-desc": "You are someone that uses DeFi and want a wallet that allows you to connect to DeFi applications",
+  "page-find-wallet-developer-title": "Developer",
+  "page-find-wallet-developer-desc": "You are developer and need a wallet to help develop and test dapps",
+  "page-find-wallet-persona-desc": "Choose the profile that matches your type of user and filter the wallet list",
+  "page-find-wallet-filters": "FILTERS",
```

Comment:
Would suggest adding this in regular casing, and use styling to make it uppercase to limit issues via translation.

Reply:
```suggestion
  "page-find-wallet-filters": "Filters",
```